src/analysis/processing/qgsalgorithmroundrastervalues.cpp

```diff
@@ -0,0 +1,262 @@
+/***************************************************************************
+                         qgsalgorithmroundrastervalues.cpp
+                         ---------------------
+    begin                : April 2020
+    copyright            : (C) 2020 by Clemens Raffler
+    email                : clemens dot raffler at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsalgorithmroundrastervalues.h"
+#include "qgsrasterfilewriter.h"
+
+///@cond PRIVATE
+
+QString QgsRoundRasterValuesAlgorithm::name() const
+{
+  return QStringLiteral( "roundrastervalues" );
+}
+
+QString QgsRoundRasterValuesAlgorithm::displayName() const
+{
+  return QObject::tr( "Round raster" );
+}
+
+QStringList QgsRoundRasterValuesAlgorithm::tags() const
+{
+  return QObject::tr( "data,cells,round,truncate" ).split( ',' );
+}
+
+QString QgsRoundRasterValuesAlgorithm::group() const
+{
+  return QObject::tr( "Raster tools" );
```

Comment:
I'm not sure Raster Tools is the right group for this. I think it fits better in the current Raster Analysis group, but ideally we'd split that group into purely analysis tools vs "manipulation" tools.

Reply:
I agree. One question on adding new processing algorithm groups between major releases: Is this considered as an API break. I plan to add more raster creation algorithms and would put them in one new group. Is it problematic to regroup algorithms/add new groups between major releases?

Reply:
It's not an API break, I think it's perfectly acceptable for a minor release!